assets/pages/contributions-landing/components/contributionsBundle.jsx

```diff
@@ -218,9 +218,11 @@ function ContributionsBundle(props: PropTypes) {
 // ----- Map State/Props ----- //
 
 function mapStateToProps(state) {
+  const contribType = (state.common.abParticipations && state.common.abParticipations.usMonthlyVsOneOff === 'monthly'
+    ? 'MONTHLY' : 'ONE_OFF') || state.page.type;
```

Comment:
As @davidfurey pointed out, setting this here unfortunately overwrites the Redux state, meaning that it stops the user from switching their contribution type.

Reply:
I do - but with this change it's still possible for me to change my contribution type...

Reply:
thanks @Ap0c  - I've changed it to do something like that now if you could take another look